azure-pipelines.yml

```diff
@@ -0,0 +1,86 @@
+trigger:
```

Comment:
Why is this file changing?

Reply:
I added this when doing some testing and carelessly left it behind. Removed.